### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by landsman**
added Nette framework implementation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ Other packages exists to integrate this SDK into the most common frameworks.
 
 - [Symfony](https://github.com/getsentry/sentry-symfony)
 - [Laravel](https://github.com/getsentry/sentry-laravel)
+- [Nette](https://github.com/Salamek/raven-nette)
```
Comment (Jean85) [inline README.md]: I would create a subsection for non-official integrations, so we could add also others like the Wordpress one or https://github.com/facile-it/sentry-module

**Comment by landsman** [comment]
Good Idea, do you will change it?

Dne 21. 6. 2017 13:18 napsal uživatel "Alessandro Lai" <
notifications@github.com>:

**Comment by Jean85** [comment]
Ok for me! Did you forget to merge? :smile:

**Comment by stayallive** [comment]
I just noticed I can't 😅  working on that 😜

**Comment by Jean85** [comment]
I did it!

**Pull request merged**